mne/beamformer/_dics.py

```diff
@@ -567,6 +568,7 @@ def apply_dics_csd(csd, filters, verbose=None):
 
     logger.info('[done]')
 
+    # XXX we should pass src to _make_stc
```

Comment:
@wmvanvliet @britta-wstnr the beamformer functions only use vertices but it's not enough to know what kind of source space you work with. There is presently an ambiguity between a standard source space with 2 hemispheres and a volume source space made of 2 labels.

do you have time to take a look?